src/libs/routes/notFoundRoute.ts

```diff
@@ -1,8 +1,8 @@
-import { Request, Response, NextFunction } from 'express'
-
-export default function notFoundRoute(req: Request, res: Response, next: NextFunction) {
+import { Request, Response, NextFunction } from 'express';
+export default (req: Request, res: Response, next: NextFunction) => {
+    console.log(`first middleWare Working`);
     next({
-        error: "Not Found",
-        code: 500
-    })
-}
\ No newline at end of file
+        err: 'Not Found',
+        code: 404
+    });
```

Comment:
use es6 and name the given function

Reply:
Updated the function name and code according to the ES6